app/src/main/res/layout/tor_one.xml

```diff
@@ -24,7 +24,7 @@
     <TextView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Neutrino Wallet does an excellent job of preventing your peers from learning of your incoming payments. However, outgoing spends are still sent sent in the clear meaning your remote peers could potentially associate your spends with your IP address. This tutorial will show you how to set up Orbot to enhance the privacy of your spends."
```

Comment:
Nit: typo in this string `...are still sent sent in the...` should be `...are still sent in the...`

Reply:
Nevermind :D I see it's fixed in the strings.xml file